tests/utils.ts

```diff
@@ -0,0 +1,64 @@
+import fs from 'node:fs/promises';
+import path from 'node:path';
+import os from 'node:os';
+
+import { execa } from 'execa';
+
+export async function createTmp() {
+  let prefix = 'v2-addon-blueprint--';
+  let prefixPath = path.join(os.tmpdir(), prefix);
+
+  let tmpDirPath = await fs.mkdtemp(prefixPath);
+
+  await fs.mkdir(tmpDirPath, { recursive: true });
+
+  return tmpDirPath;
+}
+
+/**
+  * Abstraction for install, as the blueprint supports multiple package managers
+  */
+export async function install({ cwd }: { cwd: string }) {
+  await execa('yarn', ['install', '--non-interactive'], { cwd, preferLocal: true });
+  // in order to test prepare, we need to have ignore-scripts=false
+  // this is a security risk so we'll manually invoke install + prepare
+  await execa('yarn', ['prepare'], { cwd, preferLocal: true });
```

Comment:
What is `preferLocal: true` doing here? We do run the global `yarn` here, don't we?

Reply:
correct -- I'll remove